test/infoSpec.js

```diff
@@ -7,17 +7,17 @@ const ObjStore = require('./helpers/objStore')
 const PluginVirtual = require('..')
 
 const info = {
+  currencyScale: 2,
   currencyCode: 'USD',
-  currencySymbol: '$',
   precision: 15,
-  scale: 15,
```

Comment:
should be removed everywhere as part of `mj-currency_scale` branches in different repos